config/openshift/base/300-operator_v1alpha1_openshiftpipelinesascode_crd.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: openshiftpipelinesascodes.operator.tekton.dev
+  labels:
+    version: "devel"
+    operator.tekton.dev/release: "devel"
+spec:
+  group: operator.tekton.dev
+  names:
+    kind: OpenShiftPipelinesAsCode
+    listKind: OpenShiftPipelinesAsCodeList
+    plural: openshiftpipelinesascodes
+    singular: openshiftpipelinesascode
```

Comment:
can we add shortNames as well like
```
    shortNames:
    - opac   OR
    - pac      OR
    - openshiftpac  OR anything suitable
```
Because `OpenShiftPipelinesAsCode` is big to type

Reply:
yeah make sense.

Reply:
updated!